configure.in

```diff
@@ -6487,7 +6500,7 @@ if test "$wxUSE_UIACTIONSIMULATOR" = "yes" ; then
                     AC_DEFINE(wxUSE_XTEST)
                 ],
                 [
-                    if test "$WXGTK3" = 1; then
+                    if test "$WXGTK3" = 1 -o "$WXGTK4" = 1; then
```

Comment:
It might be more clear to define `WXGTK_3_OR_4` variable once and use it rather than testing for both of those in multiple places IMO.